tokens/figma/README.md

```diff
@@ -0,0 +1,77 @@
+# Design Tokens Automation Script
+
+## Objective
+
+This script automates the process of fetching, updating, and posting design tokens to Figma for multiple brands. It reads JSON data files, extracts relevant variables, compares them with existing data from Figma, updates them if necessary, and then posts the changes back to Figma's API.
+
+## Overview of the Script
+
+### Environment Setup
+
+- The script uses the `dotenv` package to load environment variables (like Figma API tokens and file keys) from a `.env` file.
+- It imports necessary modules (`fetch`, `fs`, `path`, etc.) and utility functions (`extractJsonData`, `hexToRgba`, `extractPaletteValue`).
+
+### File and Data Preparation
+
+- The script reads JSON files from a specific directory, filters out the ones with a `.json` extension, and extracts their data using the `extractJsonData` function.
+
+### Brands and URLs
+
+- An object `brands` maps brand names to their corresponding Figma API URLs. This allows the script to handle multiple brands in one go.
```

Comment:
is this correct?

Reply:
Still correct for the update-skins script, since each file should be updated